### DISCUSSION TITLE
What is average packet loss for wifi bng (without fec applied)

### DISCUSSION THREAD

**Discussion by elderkeltir** [svpcom/wfb-ng#389] [General]
Cheers, 

I disabled fec to see the actual packet loss rate(both devices are in the same room) to understand the values.
Now I have ~3% packet loss. Is it good, is it bad? Can I somehow tune this to reduce packet loss.
I understand that fec will fix 3% loss, but still I assumed that packet loss in this conditions(wifi channel is empty, only used by me, both devices are in the same room)  should be much less.

thanks!

**Comment by svpcom** [comment]
Yes. This loss rate is normal. You can use multiple rx card to reduce it. With two cards it will be near zero.
